scripts/ci/github_actions/driver/deploy_driver.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xel
```

Comment:
since there is already operator repo checkout, can't we use these driver deployment scripts from there (instead of maintaining them twice in separate repos)?